docker/caddy/Caddyfile

```diff
@@ -14,21 +17,21 @@ route {
     root * /srv/app/public
     mercure {
         # Transport to use (default to Bolt)
-        transport_url bolt:///data/mercure.db
-        # Enable the demo endpoint (disable it in production!)
-        {$MERCURE_DEMO}
+        transport_url {$MERCURE_TRANSPORT_URL:bolt:///data/mercure.db}
         # Publisher JWT key
-        publisher_jwt {$MERCURE_PUBLISHER_JWT}
+        publisher_jwt {env.MERCURE_PUBLISHER_JWT_KEY} {env.MERCURE_PUBLISHER_JWT_ALG}
         # Subscriber JWT key
-        subscriber_jwt {$MERCURE_SUBSCRIBER_JWT}
+        subscriber_jwt {env.MERCURE_SUBSCRIBER_JWT_KEY} {env.MERCURE_SUBSCRIBER_JWT_ALG}
         # Allow anonymous subscribers (double-check that it's what you want)
         anonymous
         # Enable the subscription API (double-check that it's what you want)
         subscriptions
+        # Extra directives
+        {$MERCURE_EXTRA_DIRECTIVES}
     }
     vulcain
     push
-    php_fastcgi php:9000
-    encode gzip
+    php_fastcgi unix//var/run/php/php-fpm.sock
```

Comment:
```suggestion
    php_fastcgi unix:/var/run/php/php-fpm.sock
```

Reply:
No, it's the format expected by Caddy: https://caddyserver.com/docs/caddyfile/directives/php_fastcgi

Reply:
Oh, really. I've never seen it like this. Well, then...